lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1866,6 +1880,61 @@ derivePublicKey ctx (ApiT wid) (ApiT role_) (ApiT ix) = do
         k <- liftHandler $ W.derivePublicKey @_ @s @k @n wrk wid role_ ix
         pure $ ApiVerificationKey (getRawKey k, role_)
 
+postAnyAddress
+    :: forall n. ApiCredentials
+    -> Handler AnyAddress
+postAnyAddress body = do
```

Comment:
It looks like not just _any_ address can be posted. Is that right?

Reply:
yes, I can construct enterprise, reward account and base address for any credential (either from key or script). There is also pointer address missing that could be added in forthcoming PR. That's the rationale I proposed the name AnyAddress